pkgs/tools/security/open-fprintd/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, fetchFromGitHub, python38Packages }:
+
+python38Packages.buildPythonPackage rec {
+  pname = "open-fprintd";
+  version = "0.6";
+
+  src = fetchFromGitHub {
+    owner = "uunicorn";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-uVFuwtsmR/9epoqot3lJ/5v5OuJjuRjL7FJF7oXNDzU=";
+  };
+
+  checkInputs = with python38Packages; [ dbus-python ];
+
+  propagatedBuildInputs = with python38Packages; [ dbus-python pygobject3 ];
+
+  postInstall = ''
+    install -D -m 644 debian/open-fprintd.service \
+      $out/lib/systemd/system/open-fprintd.service
+    install -D -m 644 debian/open-fprintd-resume.service \
+      $out/lib/systemd/system/open-fprintd-resume.service
+    install -D -m 644 debian/open-fprintd-suspend.service \
+      $out/lib/systemd/system/open-fprintd-suspend.service
+    substituteInPlace $out/lib/systemd/system/open-fprintd.service \
+      --replace /usr/lib/open-fprintd "$out/lib/open-fprintd"
+    substituteInPlace $out/lib/systemd/system/open-fprintd-resume.service \
+      --replace /usr/lib/open-fprintd "$out/lib/open-fprintd"
+    substituteInPlace $out/lib/systemd/system/open-fprintd-suspend.service \
+      --replace /usr/lib/open-fprintd "$out/lib/open-fprintd"
+  '';
+
+  postFixup = ''
+    wrapPythonProgramsIn "$out/lib/open-fprintd" "$out $pythonPath"
+  '';
+
+  meta = with lib; {
+    description =
+      "Fprintd replacement which allows you to have your own backend as a standalone service";
+    homepage = "https://github.com/uunicorn/open-fprintd";
+    license = licenses.gpl2;
```

Comment:
`gpl2` is unclear, use `gpl2Plus` or `gpl2Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
Ah I see. [The package repo](https://github.com/uunicorn/open-fprintd) only contains a COPYING file, which from the instructions in the linked thread, is not sufficient to distinguish between the two. I will change it to the more restrictive `gpl2Only` until we determine otherwise.

I see that this `<license>Only` pattern is briefly mentioned in [the manual](https://nixos.org/manual/nixpkgs/stable/#var-meta-license), but isn't mentioned at all in CONTRIBUTING.md. Not sure if you see a lot of first-time requesters missing this, but I wonder if it's worth adding a little blurb that just the license name is not sufficient in one or both of those places.

Reply:
Opening issue about it in the package repo would probably be best way to find out.

I have seen that semi-often, we should probably consider updating the contributing docs. Or maybe introduce something like https://github.com/jtojnar/nixpkgs-hammering into CI pipeline, which checks for this.

Reply:
Issue created on open-fprintd: uunicorn/open-fprintd#14